en/02-javascript/0202-modelo-de-datos/index.md

```diff
@@ -228,7 +228,7 @@ Array.isArray(arr); // will be true.
 
 ### `null`
 
-There is another last value for the object type, which is `null`. This value represents the **absence of an object,** and it normally has the following uses:

```

Comment:
I spent an embarrassing amount of time trying to come up with something that didn't sound so awful. THANK YOU!